app/components/LibraryCard.js

```diff
@@ -0,0 +1,246 @@
+import React, { Component } from 'react';
+import { StyleSheet, Dimensions, View, Text, Image, TouchableOpacity, Alert } from 'react-native';
+import Ionicon from 'react-native-vector-icons/Ionicons';
+import ViewContainer from './ViewContainer';
+import { IMG_TEMP, getLibraryHours } from '../screens/LocationsList';
+import Hours from "./Hours";
+import { withNavigation } from 'react-navigation';
+
+class LibraryCard extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      collapsed: true,
+    };
+  }
+
+  handleExpandPress() {
```

Comment:
For all of the files, arrow function the function. function = () => , remove bind statements, and adjust the props param.